docs/setup/index.html

```
@@ -136,6 +137,59 @@ <h3>Persistent Install via Entrypoint</h3>
+
+        <h3>Docker Setup for Vocal Separation</h3>
+        <p>If using the automatic downloader on Linux, just ensure the variant dependencies are installed. Use the same entrypoint approach as above:</p>
+        <pre><code># docker-compose.yml
+services:
+  jellyfin:
+    image: jellyfin/jellyfin
+    entrypoint:
+      - /bin/bash
+      - -c
+      - |
+        dpkg -s libgomp1 > /dev/null 2>&1 || \
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)